src/main/c/Include/jep_platform.h

```diff
@@ -55,6 +67,80 @@ static_assert(PY_MAJOR_VERSION >= 3,"There is no Python 2 support!");
 #ifndef _Included_jep_platform
     #define _Included_jep_platform
 
+
+
+    /**
+     * **********HACK**********
+     * PyUnicode_AsUTF8 isn't part of the stable ABI until Python 3.10.
+     * See bpo-41784 and Github PR #22252: https://github.com/python/cpython/pull/22252
+     *
+     * Despite this fact, we need it for accessing the UTF8 representation of a string
+     * without any unnecessary allocations.
+     * To work around it, we define the extern function ref by hand.
+     */
+    #if defined(Py_LIMITED_API) && Py_LIMITED_API+0 < 0x030A0000
+    extern const char *PyUnicode_AsUTF8AndSize(PyObject *unicode, Py_ssize_t *size);
+    #endif
+    #if Py_LIMITED_API
+    /*
+     * ########################################
+     * API Emulation for Py_LIMITED_API.
+     * Here, we emulate some missing APIS with inline functions.
+     * These are JEP internal
+     * ########################################
+     */
+
+    /**
+     * On the limited API `PyUnicode_AsUTF8` is missing.
+     * We define it as a thin wrapper over `PyUnicode_AsUTF8AndSize`
+     */
+    static inline const char *PyUnicode_AsUTF8(PyObject *unicode) {
+        Py_ssize_t size;
+        return PyUnicode_AsUTF8AndSize(unicode, &size);
+    }
+
+    /*
+     * Emulate the 'fast sequence' API.
+     *
+     * The fast sequence API is based upon the assumption everything is a (well behaved)
+     * list or tuple. Its not hard to emulate.....
+     * Essentially `assume(PyList_Check(target) || PyTuple_Check(target))` before each call.
+     */
+
+    #undef PySequence_Fast_GET_SIZE
+    #undef PySequence_Fast_GET_ITEM
+    static inline Py_ssize_t PySequence_Fast_GET_SIZE(PyObject *target) {
+        if (PyList_Check(target) || PyTuple_Check(target)) {
+            return Py_SIZE(target); // see code
+        } else {
+            abort(); // Not a fast sequence: Undefined behavior
+        }
+    }
+    static inline PyObject *PySequence_Fast_GET_ITEM(PyObject *target, Py_ssize_t index) {
```

Comment:
Does the performance matter here?
If YES, then why are we paying the performance price of the stable ABI in other places? 
If NO, then why are we reimplementing this whole method here?